src/Uno.UWP/System/Launcher.cs

```diff
@@ -58,7 +60,11 @@ public static Task<bool> LaunchUriAsync(Uri uri)
 			}
 
 			// this method may run on the background thread on UWP
+#if !__WASM__
```

Comment:
Maybe we should have a `HasThreadAccessInternal` property (aka `HasThreadAccessButReally`), again thinking about centralization

Reply:
We're going to have to look up for other places when https://github.com/unoplatform/uno/issues/2302 will be enabled and remove #if blocks in any cases (even if we make another property for that). Let make sure we scan through properly.